browser/src/map/handler/Map.SlideShow.js

```diff
@@ -150,7 +150,8 @@ L.Map.SlideShow = L.Handler.extend({
 			}
 
 			this._slideShowWindowProxy.focus();
-			this._slideShowWindowProxy.addEventListener('keydown', this._onCloseSlideWindow);
+			this._slideShowWindowProxy.addEventListener('keydown', this._onCloseSlideWindow.bind(this));
+			this._slideShowWindowProxy.addEventListener('beforeunload', this._closeSnackbar.bind(this));
```

Comment:
Oddly, the `beforeunload` event is not working in Chrome.

Reply:
Works fine for firefox ;)

Reply:
I need your suggestion. Actually, when we close the slideshow window, we should use the `beforeunload` event to trigger an action when the window is closed. At that time, we can hide the snackbar. Unfortunately, `beforeunload` is not supported by a few older browsers, and or implementation varies across different browsers. In the previous commit code,
```js
this._slideShowWindowProxy.addEventListener('beforeunload', this._closeSnackbar.bind(this)); 
the _closeSnackbar: function(e) { this._map.uiManager.closeSnackbar(); }
```
, which works fine in Firefox but not in Chrome. I have implemented an approach that is not ideal, but it works fine. Could you please share your views or suggestions on this?